assets/static/template.go

```diff
@@ -43,16 +43,20 @@ type TemplateTranslation struct {
 	Locale_        i18n.Locale              `json:"locale"          validate:"required"`
 	Namespace_     string                   `json:"namespace"`
 	VariableCount_ int                      `json:"variable_count"`
+	Components_    []map[string]any         `json:"components"`
```

Comment:
we only use Content for generating a preview of the message so I guess components would be the same... couldn't these just be strings?

Reply:
I think we can keep using content for the preview here and make sure each template has that generate properly when syncing them

Reply:
Tho wouldn't we want to convert button components into quick replies etc? I kinda hate putting whatsapp specific knowledge in the engine, right in the send_msg action but I don't see any other way. Let's just try to isolate it in a single function for now that takes a template + parameters and returns a `MsgOut`.

The other way to approach this would be to put the onus on the UI to render a preview of the message based on the templating data.. but then the message still looks empty in API fetches, archives etc, so I think I prefer having the engine continue to create a preview.